src/components/BottomSheet/BottomSheetContentContainer/BottomSheetContentContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {memo, useCallback, useMemo, useRef} from 'react'
+import type {LayoutChangeEvent, View} from 'react-native'
+import Animated, {useAnimatedStyle, withTiming} from 'react-native-reanimated'
+import type {ITheme} from 'src/theme'
+import styled from 'styled-components/native'
+import type {BottomSheetContentContainerProps} from './types'
+
+const Container = styled(Animated.View)<{theme: ITheme}>(({theme}) => ({
+  borderTopRightRadius: theme?.sizes?.petite,
+  borderTopLeftRadius: theme?.sizes?.petite,
+}))
+
+const BottomSheetContentContainerComponent: React.FC<BottomSheetContentContainerProps> = ({
+  animatedCalculateContentHeight,
+  animatedContentHeight,
+  animatedContentTranslateY,
+
+  style,
+  children,
+}) => {
+  const containerRef = useRef<View>(null)
+
+  const animatedContentStyle = useAnimatedStyle(
+    () => ({
+      height: animatedContentHeight.value > 0 ? animatedContentHeight.value : undefined,
+      transform: [{translateY: withTiming(animatedContentTranslateY.value)}],
+    }),
+    [],
+  )
+
+  const contentStyle = useMemo(
+    () => [{backgroundColor: 'white'}, style, animatedContentStyle],
```

Comment:
backgroundColor should use from the themes

Reply:
updated